<!DOCTYPE html>
<html>
<head>
<title>Frag-MaP_protocol.md</title>
<meta http-equiv="Content-type" content="text/html;charset=UTF-8">

<style>
/* https://github.com/microsoft/vscode/blob/master/extensions/markdown-language-features/media/markdown.css */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

body {
	font-family: var(--vscode-markdown-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif);
	font-size: var(--vscode-markdown-font-size, 14px);
	padding: 0 26px;
	line-height: var(--vscode-markdown-line-height, 22px);
	word-wrap: break-word;
}

#code-csp-warning {
	position: fixed;
	top: 0;
	right: 0;
	color: white;
	margin: 16px;
	text-align: center;
	font-size: 12px;
	font-family: sans-serif;
	background-color:#444444;
	cursor: pointer;
	padding: 6px;
	box-shadow: 1px 1px 1px rgba(0,0,0,.25);
}

#code-csp-warning:hover {
	text-decoration: none;
	background-color:#007acc;
	box-shadow: 2px 2px 2px rgba(0,0,0,.25);
}

body.scrollBeyondLastLine {
	margin-bottom: calc(100vh - 22px);
}

body.showEditorSelection .code-line {
	position: relative;
}

body.showEditorSelection .code-active-line:before,
body.showEditorSelection .code-line:hover:before {
	content: "";
	display: block;
	position: absolute;
	top: 0;
	left: -12px;
	height: 100%;
}

body.showEditorSelection li.code-active-line:before,
body.showEditorSelection li.code-line:hover:before {
	left: -30px;
}

.vscode-light.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(0, 0, 0, 0.15);
}

.vscode-light.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(0, 0, 0, 0.40);
}

.vscode-light.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-dark.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 255, 255, 0.4);
}

.vscode-dark.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 255, 255, 0.60);
}

.vscode-dark.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-high-contrast.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 160, 0, 0.7);
}

.vscode-high-contrast.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 160, 0, 1);
}

.vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

img {
	max-width: 100%;
	max-height: 100%;
}

a {
	text-decoration: none;
}

a:hover {
	text-decoration: underline;
}

a:focus,
input:focus,
select:focus,
textarea:focus {
	outline: 1px solid -webkit-focus-ring-color;
	outline-offset: -1px;
}

hr {
	border: 0;
	height: 2px;
	border-bottom: 2px solid;
}

h1 {
	padding-bottom: 0.3em;
	line-height: 1.2;
	border-bottom-width: 1px;
	border-bottom-style: solid;
}

h1, h2, h3 {
	font-weight: normal;
}

table {
	border-collapse: collapse;
}

table > thead > tr > th {
	text-align: left;
	border-bottom: 1px solid;
}

table > thead > tr > th,
table > thead > tr > td,
table > tbody > tr > th,
table > tbody > tr > td {
	padding: 5px 10px;
}

table > tbody > tr + tr > td {
	border-top: 1px solid;
}

blockquote {
	margin: 0 7px 0 5px;
	padding: 0 16px 0 10px;
	border-left-width: 5px;
	border-left-style: solid;
}

code {
	font-family: Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback";
	font-size: 1em;
	line-height: 1.357em;
}

body.wordWrap pre {
	white-space: pre-wrap;
}

pre:not(.hljs),
pre.hljs code > div {
	padding: 16px;
	border-radius: 3px;
	overflow: auto;
}

pre code {
	color: var(--vscode-editor-foreground);
	tab-size: 4;
}

/** Theming */

.vscode-light pre {
	background-color: rgba(220, 220, 220, 0.4);
}

.vscode-dark pre {
	background-color: rgba(10, 10, 10, 0.4);
}

.vscode-high-contrast pre {
	background-color: rgb(0, 0, 0);
}

.vscode-high-contrast h1 {
	border-color: rgb(0, 0, 0);
}

.vscode-light table > thead > tr > th {
	border-color: rgba(0, 0, 0, 0.69);
}

.vscode-dark table > thead > tr > th {
	border-color: rgba(255, 255, 255, 0.69);
}

.vscode-light h1,
.vscode-light hr,
.vscode-light table > tbody > tr + tr > td {
	border-color: rgba(0, 0, 0, 0.18);
}

.vscode-dark h1,
.vscode-dark hr,
.vscode-dark table > tbody > tr + tr > td {
	border-color: rgba(255, 255, 255, 0.18);
}

</style>

<style>
/* Tomorrow Theme */
/* http://jmblog.github.com/color-themes-for-google-code-highlightjs */
/* Original theme - https://github.com/chriskempson/tomorrow-theme */

/* Tomorrow Comment */
.hljs-comment,
.hljs-quote {
	color: #8e908c;
}

/* Tomorrow Red */
.hljs-variable,
.hljs-template-variable,
.hljs-tag,
.hljs-name,
.hljs-selector-id,
.hljs-selector-class,
.hljs-regexp,
.hljs-deletion {
	color: #c82829;
}

/* Tomorrow Orange */
.hljs-number,
.hljs-built_in,
.hljs-builtin-name,
.hljs-literal,
.hljs-type,
.hljs-params,
.hljs-meta,
.hljs-link {
	color: #f5871f;
}

/* Tomorrow Yellow */
.hljs-attribute {
	color: #eab700;
}

/* Tomorrow Green */
.hljs-string,
.hljs-symbol,
.hljs-bullet,
.hljs-addition {
	color: #718c00;
}

/* Tomorrow Blue */
.hljs-title,
.hljs-section {
	color: #4271ae;
}

/* Tomorrow Purple */
.hljs-keyword,
.hljs-selector-tag {
	color: #8959a8;
}

.hljs {
	display: block;
	overflow-x: auto;
	color: #4d4d4c;
	padding: 0.5em;
}

.hljs-emphasis {
	font-style: italic;
}

.hljs-strong {
	font-weight: bold;
}
</style>

<style>
/*
 * Markdown PDF CSS
 */

 body {
	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif, "Meiryo";
	padding: 0 12px;
}

pre {
	background-color: #f8f8f8;
	border: 1px solid #cccccc;
	border-radius: 3px;
	overflow-x: auto;
	white-space: pre-wrap;
	overflow-wrap: break-word;
}

pre:not(.hljs) {
	padding: 23px;
	line-height: 19px;
}

blockquote {
	background: rgba(127, 127, 127, 0.1);
	border-color: rgba(0, 122, 204, 0.5);
}

.emoji {
	height: 1.4em;
}

code {
	font-size: 14px;
	line-height: 19px;
}

/* for inline code */
:not(pre):not(.hljs) > code {
	color: #C9AE75; /* Change the old color so it seems less like an error */
	font-size: inherit;
}

/* Page Break : use <div class="page"/> to insert page break
-------------------------------------------------------- */
.page {
	page-break-after: always;
}

</style>

<script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
</head>
<body>
  <script>
    mermaid.initialize({
      startOnLoad: true,
      theme: document.body.classList.contains('vscode-dark') || document.body.classList.contains('vscode-high-contrast')
          ? 'dark'
          : 'default'
    });
  </script>
<h1 id="frag-map-experimental-protocol">Frag-MaP experimental protocol</h1>
<p>Seth D. Veenbaas
10/16/2023</p>
<h2 id="experiment-summary">Experiment Summary</h2>
<p>Detect the RNA binding site of fully-functionalized fragments in B. subtilis total RNA.</p>
<h2 id="protocol-overview">Protocol Overview:</h2>
<ul>
<li>Photocrosslink and extract RNA</li>
<li>Partial Fragmentation of total RNA</li>
<li>Click Azide-PEG3-Biotin</li>
<li>Streptavidin bead enrichment (optional)</li>
<li>Ethanol precipitation</li>
<li>MaP RT</li>
<li>Second strand synthesis</li>
<li>NEB Ultra II adapter ligation kit</li>
<li>Qubit</li>
<li>Bioanalyzer</li>
<li>Pool and dilute library</li>
<li>Miseq</li>
</ul>
<h2 id="photocrosslink-and-extract-rna-3-days">Photocrosslink and extract RNA (3 days)</h2>
<p>Prepare ligand crosslinked total RNA for JuMP RT, and sequencing.</p>
<h3 id="note-all-experiments-should-start-fresh-by-streaking-b-sub-on-plates"><strong>NOTE: All experiments should start fresh by streaking B. sub on plates.</strong></h3>
<ul>
<li>
<p>B. sub does not grow well from glycerol stocks or passaged cell pellets.</p>
</li>
<li>
<p>B. sub does not store well at 4 °C because it enters a spore form.</p>
</li>
</ul>
<h3 id="day-1-plate-cells">Day 1: Plate cells</h3>
<h4 id="materials">Materials:</h4>
<ul>
<li>B. subtilis str. 168 glycerol stock</li>
<li>LB Agar plate</li>
</ul>
<h4 id="protocol">Protocol:</h4>
<ol>
<li>Streak a plate with B. subtilis str. 168 to form single colonies.</li>
<li>Incubate plate at 30 °C overnight.</li>
</ol>
<h3 id="day-2-overnight-culture">Day 2: Overnight culture</h3>
<h4 id="materials">Materials:</h4>
<ul>
<li>LB media (5 mL)</li>
<li>Culture tube</li>
</ul>
<h4 id="protocol">Protocol:</h4>
<ol>
<li>Inoculate 5 mL LB media with a single colony from plate.</li>
<li>Grow cells overnight at 30 °C, 225 rpm.</li>
</ol>
<h3 id="day-3-probe-cells-and-extract-rna-10-hours">Day 3: Probe cells and extract RNA (~10 hours)</h3>
<h4 id="materials">Materials:</h4>
<ul>
<li>LB media (50 mL)</li>
<li>PBS buffer (100 mL)</li>
<li>50 mM ligand stock in DMSO (20 uL per sample)</li>
<li>Sterile 6-well plastic plate (1 well per sample)</li>
<li>lysozyme (60 mg)</li>
<li>1M Tris pH 7 (180 uL)</li>
<li>0.5 M EDTA (120 uL)</li>
<li>Chloroform</li>
<li>100% ethanol</li>
<li>Qiogen RNAeasy Plus Universial Mini kit</li>
</ul>
<h4 id="protocol">Protocol:</h4>
<ol start="3">
<li>
<p>Inoculate 60 mL LB media with 3 mL cultured cells and grow cells to 0.50 OD600.</p>
<p><strong>NOTE: Expect cell growth to take ~2.5 hours.</strong></p>
<table>
<thead>
<tr>
<th style="text-align:center">Time point</th>
<th style="text-align:center">#1 OD600</th>
<th style="text-align:center">#2 OD600</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center">-</td>
<td style="text-align:center">-</td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Buffer excahnge cells.</p>
<ul>
<li>Gently pellet cells at 3200g for 10 min at 4 °C.</li>
<li>Carefully remove supernatant.</li>
<li>Wash cell pellet with 50 mL PBS buffer, repeat centrifugation, and carefully remove supernatant.</li>
<li>Resuspend cells in 50 mL PBS buffer.</li>
</ul>
</li>
<li>
<p>Probe cells with functionalized ligands.</p>
<ul>
<li>
<p>Combine 4,980 uL of cultured media with 20 uL of photo-reactive ligand (50 mM stock in DMSO) and pipette mix. Avoid prolonged light exposure.</p>
<p><strong>NOTE: final [ligand] is 200 uM. 250X dilution.</strong></p>
</li>
<li>
<p>Incubate ligand-treated cells for 30 minutes at 30 °C and 225 rpm.</p>
<p><strong>NOTE: Keep solution in dark</strong>.</p>
</li>
</ul>
</li>
<li>
<p>Cross-link functionalized ligands to RNA.</p>
<ul>
<li>
<p>Transfer 5 mL cell solution to a sterile 6-well plastic plate.</p>
</li>
<li>
<p>Place uncover plate and place on ice</p>
</li>
<li>
<p>Irradiate with 365 nm light source and cross-link with 3 J/cm2.</p>
<p><strong>NOTE: This is nine minutes (or three max energy hits) 5-6 cm from the light source in a UVP crosslinker.</strong></p>
</li>
</ul>
</li>
<li>
<p>Create cell pellet.</p>
<ul>
<li>Centrifuge to pellet the cells. 8,000xg for 10 min. at 4 °C.</li>
<li>Discard supernatant.</li>
</ul>
</li>
<li>
<p>Prepare fresh lysis buffer.</p>
<ul>
<li>(30 mM Tris pH 7, 10 mM EDTA, 10 mg/mL lysozyme):</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Reagent</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">lysozyme</td>
<td style="text-align:left">50 mg</td>
</tr>
<tr>
<td style="text-align:left">1M Tris pH 7</td>
<td style="text-align:left">150 uL</td>
</tr>
<tr>
<td style="text-align:left">0.5M EDTA</td>
<td style="text-align:left">100 uL</td>
</tr>
<tr>
<td style="text-align:left">RNase inhibitor murine</td>
<td style="text-align:left">50 uL</td>
</tr>
<tr>
<td style="text-align:left">NF H2O</td>
<td style="text-align:left">4700 uL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>5,000 uL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Lysis cell pellet with lysozyme buffer.</p>
<ul>
<li>Add 250 uL of buffer lysis to each cell pellet.</li>
<li>Incubate for 30 min. at RT.</li>
</ul>
</li>
<li>
<p>Extract and purify total RNA from cells.</p>
<ul>
<li>
<p>Use 1000 uL of Tri-reagent</p>
</li>
<li>
<p>Mix aggressively to aid in lysis</p>
</li>
<li>
<p>Lyse for 15 minutes</p>
</li>
<li>
<p>Add 250 uL of chloroform</p>
</li>
<li>
<p>Vortex and incubate at RT for 3-5 minutes</p>
</li>
<li>
<p>Centrifuge at 15000xg for 15 minutes at 4 C</p>
</li>
<li>
<p><strong>NOTE: Keep sample on ice until aqueous layer is recovered</strong></p>
</li>
<li>
<p>Add 100% ethanol to each recovered aqueous layer in equal parts (1:1)</p>
</li>
<li>
<p>Follow Direct-zol RNA Miniprep Plus Kit instructions.</p>
</li>
<li>
<p>Elute from column with 50 uL of NFW</p>
<p><strong>NOTE: Yields of ~50-60 ug total RNA have been achieved per 5 mL culture probed.</strong></p>
</li>
</ul>
</li>
<li>
<p>Check RNA purity.</p>
<ul>
<li>Measure absorbance ratio (A260/280 and A260/230) on nanodrop.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:center">Culture</th>
<th style="text-align:center">Sample ID</th>
<th style="text-align:center">Yield (μg)</th>
<th style="text-align:center">[RNA] (ng/μL)</th>
<th style="text-align:center">A260/280</th>
<th>A260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td></td>
</tr>
</tbody>
</table>
</li>
</ol>
<h3 id="notes">NOTES:</h3>
<ul>
<li>Samples diluted to 500 ng/μL and flash frozen.</li>
</ul>
<h3 id="storage">Storage:</h3>
<ul>
<li>RNA can be stored overnight at 4 °C for ~ 1 week.</li>
<li>RNA can be flash-frozen in aliquots and stored at -80 °C for ~1 year.</li>
</ul>
<h3 id="next-steps">Next Steps:</h3>
<ul>
<li>Partial RNA Fragmentation</li>
<li>MaP-RT</li>
</ul>
<h2 id="sample-details">Sample Details:</h2>
<p>RNA provided from Frag-MaP Cell Probing 4 experiment (200 μM).</p>
<table>
<thead>
<tr>
<th style="text-align:left">Sample ID</th>
<th style="text-align:center">Replicate</th>
<th style="text-align:center">Ligand</th>
<th style="text-align:center">Fragmention</th>
<th style="text-align:center">Enrichment</th>
<th style="text-align:center">[RNA] (ng/μL)</th>
<th style="text-align:center">i7 index</th>
<th style="text-align:center">i5 index</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
</tbody>
</table>
<h2 id="partial-fragmentation-of-total-rna-05-hour">Partial Fragmentation of total RNA (~0.5 hour)</h2>
<p>Fragment RNA to around 150 nt average length.</p>
<h3 id="materials">Materials:</h3>
<ul>
<li>Total RNA (10 μg)</li>
<li><a href="https://www.neb.com/products/e6150-nebnext-magnesium-rna-fragmentation-module#Product%20Information">NEBNext® Magnesium RNA Fragmentation Module</a></li>
</ul>
<h3 id="protocol">Protocol:</h3>
<ol>
<li>Mix the following components in a sterile PCR tube:</li>
</ol>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:right">Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Purified total RNA (~10 μg)</td>
<td style="text-align:right">18 μL</td>
</tr>
<tr>
<td style="text-align:left">RNA Fragmentation Buffer (10X)</td>
<td style="text-align:right">2 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>Total</strong></td>
<td style="text-align:right"><strong>20 μL</strong></td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Incubate in a preheated thermal cycler for 3 minutes at 94°C.</li>
<li>Transfer tube to ice immediately.</li>
<li>Add 2 μl 10X RNA Fragmentation Stop Solution.</li>
<li>Clean up with <a href="https://www.neb.com/en-us/products/t2030-monarch-rna-cleanup-kit-10-ug#Product%20Information">Monarch RNA Cleanup kit</a></li>
</ol>
<h3 id="more-information">More Information:</h3>
<p><a href="https://www.neb.com/protocols/0001/01/01/nebnext-magnesium-rna-fragmentation-module-protocol-e6150">NEBNext® Magnesium RNA Fragmentation Module Protocol</a></p>
<h2 id="click-azide-peg3-biotin-1-hour">Click Azide-PEG3-Biotin (~1 hour)</h2>
<p>This prtocol is intented to add a vector to enable biotin-streptavin enrichment for crosslinked RNA.</p>
<h3 id="materials">Materials:</h3>
<p>Multiple reagents amounts by the number of intended samples</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">[Reagent]</th>
<th style="text-align:left">[Final]</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Total RNA-alkyne – 10X</td>
<td style="text-align:left">~10 μg</td>
<td style="text-align:left">~0.1 μM</td>
<td style="text-align:left">18 μL</td>
</tr>
<tr>
<td style="text-align:left">N3-Cy5 – 25X</td>
<td style="text-align:left">50 μM</td>
<td style="text-align:left">2 μM</td>
<td style="text-align:left">2 μL</td>
</tr>
<tr>
<td style="text-align:left">HEPES Buffer pH 7.5 – 25X</td>
<td style="text-align:left">250 mM</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">2 μL</td>
</tr>
<tr>
<td style="text-align:left">DMSO</td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left">25 μL</td>
</tr>
<tr>
<td style="text-align:left">Ascorbic Acid – 20X</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">0.5 mM</td>
<td style="text-align:left">2.5 μL</td>
</tr>
<tr>
<td style="text-align:left">Cu(II): TBTA – 41.7X</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">240 μM</td>
<td style="text-align:left">1.2 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left"><strong>50 μL</strong></td>
</tr>
</tbody>
</table>
<h3 id="protocol">Protocol:</h3>
<ol>
<li>
<p>Prepare fresh 10 mM Ascorbic acid stock 20X:</p>
<ul>
<li>Combine reagents.</li>
<li>Sparge solution with N2 and use immediately.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Ascorbic acid</td>
<td style="text-align:left">9.0 mg</td>
</tr>
<tr>
<td style="text-align:left">NF Water</td>
<td style="text-align:left">5.0 mL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>5.0 mL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Dilute RNA:</p>
<ul>
<li>Add RNA-alkyne and water to a strip of PCR tubes.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">[Reagent]</th>
<th style="text-align:left">Final [RNA-Alkyne]</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">RNA-alkyne – 10X</td>
<td style="text-align:left">75 ng/μL</td>
<td style="text-align:left">~ 0.01 μM</td>
<td style="text-align:left">17.3 μL</td>
</tr>
<tr>
<td style="text-align:left">Nuclease free H2O</td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left">- μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>Subtotal</strong></td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left"><strong>17.3 μL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Heat denature RNA:</p>
<ul>
<li>Incubate RNA at 95 °C for 2 min.</li>
<li>Snap cool on ice for 2 min.</li>
</ul>
</li>
<li>
<p>Add HEPES buffer and N3-PEG3-Biotin to tubes:</p>
<ul>
<li>Add HEPES and N3-PEG3-Biotin.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">[Reagent]</th>
<th style="text-align:left">[Final]</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">HEPES Buffer pH 7.5 – 25X</td>
<td style="text-align:left">250 mM</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">2 μL</td>
</tr>
<tr>
<td style="text-align:left">N3-PEG3-Biotin – 25X</td>
<td style="text-align:left">50 μM</td>
<td style="text-align:left">2 μM</td>
<td style="text-align:left">2 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>Subtotal</strong></td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left"><strong>21.3 μL</strong></td>
</tr>
</tbody>
</table>
<p><strong>NOTE: Batch samples can be kept on ice after HEPES addition.</strong></p>
</li>
<li>
<p>Denature RNA by adding DMSO:</p>
<ul>
<li>
<p>Add DMSO to each sample.</p>
</li>
<li>
<p>Briefly vortex.</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">[Reagent]</th>
<th style="text-align:left">[Final]</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">DMSO</td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left">25 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>Subtotal</strong></td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left"><strong>46.3 μL</strong></td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
<li>
<p>Start click reaction:</p>
<ul>
<li>
<p>Add ascorbic acid and Cu(II)/TBTA mix to samples.</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagents</th>
<th style="text-align:left">[Reagent]</th>
<th style="text-align:left">[Final]</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Ascorbic Acid – 20X</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">0.5 mM</td>
<td style="text-align:left">2.5 μL</td>
</tr>
<tr>
<td style="text-align:left">Cu(II): TBTA – 41.7X</td>
<td style="text-align:left">10 mM</td>
<td style="text-align:left">240 uM</td>
<td style="text-align:left">1.2 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>Total</strong></td>
<td style="text-align:left">-</td>
<td style="text-align:left">-</td>
<td style="text-align:left"><strong>50.0 μL</strong></td>
</tr>
</tbody>
</table>
<p><strong>NOTE: Initiate reaction within 5 minutes of adding DMSO (RNA stays denatured for ~30 minutes).</strong></p>
</li>
</ul>
</li>
<li>
<p>Incubate reaction.</p>
<ul>
<li>
<p>Cover reaction.</p>
</li>
<li>
<p>Incubate at 40 °C for ~20 minutes.</p>
<p><strong>NOTE: No further conversion after 1 h and more RNA degradation occurs.</strong></p>
</li>
</ul>
</li>
<li>
<p>Quench reaction.</p>
<ul>
<li>Add 1.0 μL of 0.5 M EDTA to each sample (50 μL).</li>
</ul>
</li>
<li>
<p>Remove DMSO.</p>
<ul>
<li>Dilute to ~500 μL with NF water (final [DMSO] is 5%).</li>
<li>Concentrate with Amicon Ultra 0.5mL Centrifugal filter spin column (10-50K NMWL cut off). Spin for 10 min at 14,000 g and 4 °C.</li>
</ul>
<p><strong>OPTIONAL:</strong></p>
<ul>
<li>Wash column with ~500 μL of cold TE buffer</li>
<li>Spin for 10 min at 14,000 g and 4 °C.</li>
</ul>
</li>
</ol>
<h3 id="results---nanodrop">Results - Nanodrop:</h3>
<table>
<thead>
<tr>
<th style="text-align:left">Sample ID</th>
<th style="text-align:center">[RNA] (ng/μL)</th>
<th style="text-align:center">A260/280</th>
<th style="text-align:center">A260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">1D</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1M</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1Z</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1Q</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1B</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1H</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1G</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">1J</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2D</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2M</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2Z</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2Q</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2B</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2H</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2G</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left">2J</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
</tbody>
</table>
<h2 id="streptavidin-bead-enrichment-1-hour">Streptavidin bead enrichment (~1 hour)</h2>
<p>Enrich crosslinked RNA via Biotin-Streptavidin pull-down.</p>
<h3 id="before-starting">Before starting:</h3>
<ul>
<li>click Azide-3PEG-biotin to Alkyne-RNA and purify (RNAeasy Minelute)</li>
</ul>
<h3 id="materials">Materials:</h3>
<ul>
<li>0.5M Tris pH 7.5</li>
<li>2M NaCl</li>
<li>10% Tween-20</li>
<li><a href="https://www.neb.com/products/s1420-streptavidin-magnetic-beads#Product%20Information">Streptavidin Magnetic Beads</a></li>
<li>Formamide</li>
<li>0.5M EDTA</li>
</ul>
<h3 id="protocol">Protocol:</h3>
<ol>
<li>
<p>Prepare Bind Buffer (2X):</p>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:center">[reagant]</th>
<th style="text-align:center">[2X]</th>
<th style="text-align:center">[Final]</th>
<th style="text-align:right">Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Tris pH 7.5</td>
<td style="text-align:center">1 M</td>
<td style="text-align:center">100 mM</td>
<td style="text-align:center">50mM</td>
<td style="text-align:right">1,000</td>
</tr>
<tr>
<td style="text-align:left">KCl</td>
<td style="text-align:center">2 M</td>
<td style="text-align:center">200 mM</td>
<td style="text-align:center">100 mM</td>
<td style="text-align:right">2,000</td>
</tr>
<tr>
<td style="text-align:left">TWEEN-20</td>
<td style="text-align:center">10%</td>
<td style="text-align:center">0.02%</td>
<td style="text-align:center">0.01%</td>
<td style="text-align:right">20</td>
</tr>
<tr>
<td style="text-align:left">NFW</td>
<td style="text-align:center">-</td>
<td style="text-align:center">-</td>
<td style="text-align:center">-</td>
<td style="text-align:right">6,980</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:right"><strong>10,000</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Prepare Wash Buffer:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:center">[reagant]</th>
<th style="text-align:right">[Final]</th>
<th style="text-align:right">Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Bind Buffer</td>
<td style="text-align:center">2X</td>
<td style="text-align:right">1X</td>
<td style="text-align:right">7,000</td>
</tr>
<tr>
<td style="text-align:left">NFW</td>
<td style="text-align:center">-</td>
<td style="text-align:right">-</td>
<td style="text-align:right">7,000</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:right"><strong>-</strong></td>
<td style="text-align:right"><strong>14,000</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Prepare Elution Buffer:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:center">Concentration</th>
<th style="text-align:center">[Final]</th>
<th style="text-align:right">Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Formamide</td>
<td style="text-align:center">99%</td>
<td style="text-align:center">95%</td>
<td style="text-align:right">960</td>
</tr>
<tr>
<td style="text-align:left">EDTA (pH 8.2)</td>
<td style="text-align:center">0.5M</td>
<td style="text-align:center">10 mM</td>
<td style="text-align:right">20</td>
</tr>
<tr>
<td style="text-align:left">NFW</td>
<td style="text-align:center">-</td>
<td style="text-align:center">-</td>
<td style="text-align:right">20</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:right"><strong>1000</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Wash NEB Streptavidin Magnatic Beads.</p>
<ul>
<li>Vortex beads until comletely suspended.</li>
<li>Add 50 μL of beads to a PCR strip.</li>
<li>Incubaute at RT for 5 minutes.</li>
<li>Apply magnet to beads and remove supernatant.</li>
<li>Resuspend beads in 100 μL of Wash Buffer (1X).</li>
<li>Apply magnet to beads and remove supernatant.</li>
<li>Repeat wash with Wash Buffer (1X).</li>
</ul>
</li>
<li>
<p>Bind RNA to beads:</p>
<ul>
<li>Add RNA-biotin to a strip of PCR tubes.</li>
<li>Add Bind Buffer (2X).</li>
<li>Add NEB Streptavidin Magnatic Beads.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:center">Concentration</th>
<th style="text-align:center">[Final]</th>
<th style="text-align:right">Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">RNA-Biotin</td>
<td style="text-align:center">-</td>
<td style="text-align:center">5 μg</td>
<td style="text-align:right">25</td>
</tr>
<tr>
<td style="text-align:left">Bind Buffer</td>
<td style="text-align:center">2X</td>
<td style="text-align:center">1X</td>
<td style="text-align:right">25</td>
</tr>
<tr>
<td style="text-align:left">Streptavidin Beads</td>
<td style="text-align:center">-</td>
<td style="text-align:center">-</td>
<td style="text-align:right">50</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:center"><strong>-</strong></td>
<td style="text-align:right"><strong>100</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Incubate RNA-Biotin and Streptavidin Beads for at least 30 minutes at RT.</p>
<ul>
<li>&lt; 10 minutes for short oligios (~ 30 nucleotides).</li>
<li>~ 15 minutes for fragmented total RNA.</li>
<li>~ 30 minutes for full length total RNA.</li>
</ul>
</li>
<li>
<p>Pull-down:</p>
<ul>
<li>Immobilize beads with magnetic rack.</li>
<li>Remove and discard supernatant.</li>
</ul>
</li>
<li>
<p>Wash beads 4 times:</p>
<ul>
<li>Resuspend beads in 150 μL of Wash Buffer (1X).</li>
<li>Immobilize beads with magnetic rack.</li>
<li>Remove and discard supernatant.</li>
</ul>
</li>
<li>
<p>Elute RNA:</p>
<ul>
<li>Add 50 μL of Elution Buffer.</li>
<li>Heat tubes at 95 °C for 4 minutes.</li>
<li>Quickly transfer tubes to magnetic rack and immobilize beads.</li>
<li>Transfer RNA-biotin solution into a new tube.</li>
</ul>
</li>
</ol>
<h3 id="more-information">More information:</h3>
<ul>
<li><a href="https://www.nature.com/articles/s41467-021-25973-x"><em>A chemical probe based on the PreQ1 metabolite enables transcriptome-wide mapping of binding sites</em></a></li>
</ul>
<h3 id="next-steps">Next steps:</h3>
<ul>
<li><a href="ethanol-rna-precipitation.md">Ethanol precipitation</a></li>
</ul>
<h2 id="purify-rna---ethanol-precipitation-2-hours">Purify RNA - Ethanol Precipitation (~2 hours)</h2>
<p>Purified RNA may need to be concentrated by precipitation for downstream
applications. Precipitation of RNA with ethanol (or isopropanol) is the
standard method to recover RNA from aqueous solutions.</p>
<h3 id="materials">Materials:</h3>
<ul>
<li>
<p>RNA (&gt; 10 ng/μL)*</p>
</li>
<li>
<p>100% Ethanol (or isopropanol)</p>
</li>
<li>
<p>70% Ethanol</p>
</li>
<li>
<p>A salt from TABLE 1.</p>
<p>TABLE 1. Salt solutions for precipitating RNA</p>
<table>
<thead>
<tr>
<th style="text-align:left">Salt</th>
<th style="text-align:right">Stock (M)</th>
<th style="text-align:right">[Final] (M)</th>
<th>Comments</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Sodium Acetate</td>
<td style="text-align:right">3 (pH 5.2)</td>
<td style="text-align:right">0.3</td>
<td>Recommended.</td>
</tr>
<tr>
<td style="text-align:left">Ammonium acetate</td>
<td style="text-align:right">5</td>
<td style="text-align:right">0.5</td>
<td>Use with isopropanol to remove free nucleotides.</td>
</tr>
<tr>
<td style="text-align:left">Lithium Chloride</td>
<td style="text-align:right">8</td>
<td style="text-align:right">0.8</td>
<td>Avoid for downstream IVT or RT.</td>
</tr>
</tbody>
</table>
</li>
</ul>
<p><strong>NOTE: Avoid Potassium salts if RNA will be exposed to dodecyl sulfate (SDS).</strong></p>
<h3 id="protocol">Protocol:</h3>
<ol>
<li>
<p>Combined components:</p>
<ul>
<li>Add RNA to appropriately sized tube.</li>
<li>Add one of the salt solutions shown in Table 1.</li>
<li>Add 2.5–3.0 volumes of ice-cold ethanol (or 1 volume of isopropanol).</li>
<li>Mix the solution well.</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">Component</th>
<th style="text-align:right">Concentration</th>
<th style="text-align:right">Equivalents</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">RNA</td>
<td style="text-align:right">&gt;10 ng/μL*</td>
<td style="text-align:right">1</td>
</tr>
<tr>
<td style="text-align:left">Salt</td>
<td style="text-align:right">10X</td>
<td style="text-align:right">0.1</td>
</tr>
<tr>
<td style="text-align:left">Ethanol</td>
<td style="text-align:right">100%</td>
<td style="text-align:right">2.9</td>
</tr>
<tr>
<td style="text-align:left">Glycogen (*optional)</td>
<td style="text-align:right">100X</td>
<td style="text-align:right">-</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:right"><strong>-</strong></td>
<td style="text-align:right"><strong>4</strong></td>
</tr>
</tbody>
</table>
<p>*If [RNA] is &lt; 10 ng/μL, add glycogen (final concentration of 50–150 μg/mL)
to facilitate low concentration precipitation.</p>
</li>
<li>
<p>Precipitate RNA:</p>
<ul>
<li>Store the ethanolic solution for 1 h to overnight at −20 °C.</li>
<li>RNA precipitation is faster and more complete at higher RNA concentrations.</li>
</ul>
</li>
<li>
<p>Recover the RNA:</p>
<ul>
<li>Centrifugation RNA solution at 12,000g–14,000g for 10 min at 4 °C to form pellet.</li>
<li>RNA pellet is is often invisible.</li>
<li>Decant the supernatant without disturbing the RNA pellet.</li>
<li>Carefully remove remaining traces of the supernatant with a disposable pipette tip.</li>
</ul>
</li>
<li>
<p>Wash the RNA pellet (x2):</p>
<ul>
<li>Add 0.5 mL of ice-cold 70% ethanol to pellet.</li>
<li>Centrifuge at maximum speed for 10min at 4 °C.</li>
<li>Repeat ethanol wash.</li>
</ul>
</li>
<li>
<p>Dry RNA pellet:</p>
<ul>
<li>Leave tube open at room temperature until the last traces of fluid have evaporated.</li>
<li>Do not dry the RNA pellet completely; otherwise, it can be difficult to dissolve the RNA.</li>
</ul>
</li>
<li>
<p>Dissolve the RNA pellet:</p>
<ul>
<li>Dissolve RNA in the desired volume of an RNase-free buffer (usually TE; pH between 6 and 7).</li>
<li>Rinse the walls of the tube well with the buffer.</li>
</ul>
</li>
</ol>
<h2 id="storage">Storage:</h2>
<ul>
<li>RNA can be stored for up to 1 yr at −80 °C.</li>
</ul>
<h2 id="more-information">More information:</h2>
<ul>
<li><a href="https://cshprotocols.cshlp.org/content/2020/3/pdb.prot101717.full">Precipitation of RNA with Ethanol</a></li>
</ul>
<h1 id="mutational-profiling-rt-3-hours">Mutational profiling RT (~3 hours)</h1>
<p>This is protocol is based on Smola et.al., 2015. It has been optimized for the
synthesis of long cDNA from DMS modified RNA.</p>
<h2 id="before-starting">Before starting:</h2>
<p><a href="../chemical-reactions/">Modify RNA</a>
<a href="../purification/">Purify RNA</a></p>
<h2 id="materials">Materials:</h2>
<ul>
<li>0.5 M Tris pH 8.0</li>
<li>0.75 M KCl</li>
<li>0.1 M DTT</li>
<li>RNA (1 ng to 5 μg of total RNA)</li>
<li>2 µM RT primer (50 to 250 ng random primer)</li>
<li>10 mM dNTP mix</li>
<li>5 M betaine</li>
<li>40 mM MnCl2</li>
<li>SuperScript II</li>
</ul>
<h2 id="protocol">Protocol:</h2>
<ol>
<li>
<p>Create fresh 10X NTP minus:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagent</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">1 M Tris ph 8.0</td>
<td style="text-align:left">20 uL</td>
</tr>
<tr>
<td style="text-align:left">2 M KCl</td>
<td style="text-align:left">15 uL</td>
</tr>
<tr>
<td style="text-align:left">1 M DTT</td>
<td style="text-align:left">4 uL</td>
</tr>
<tr>
<td style="text-align:left">NF Water</td>
<td style="text-align:left">1 uL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>40 uL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>For each RNA sample combine:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagent</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">RNA (~15 ng)</td>
<td style="text-align:left">7.5 uL</td>
</tr>
<tr>
<td style="text-align:left">nonamer (~100 ng)</td>
<td style="text-align:left">0.5 uL</td>
</tr>
<tr>
<td style="text-align:left">10 mM dNTPs</td>
<td style="text-align:left">2 uL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>10 µL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Denature and anneal ribosome by incubating at 90°C for 5 min, then cool on ice.</p>
</li>
<li>
<p>Create the following master mix:</p>
<p><em><strong>(multiply by number of samples+1)</strong></em></p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagent</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">10X NTP minus</td>
<td style="text-align:left">2 uL</td>
</tr>
<tr>
<td style="text-align:left">5 M Betaine</td>
<td style="text-align:left">4 uL</td>
</tr>
<tr>
<td style="text-align:left">40 mM MnCl2</td>
<td style="text-align:left">3 uL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>9 uL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Add 9 µL of master mix to each sample.</p>
</li>
<li>
<p>Incubate at 25°C for 2 min.</p>
</li>
<li>
<p>Add 1 µL of SuperScript II.</p>
</li>
<li>
<p>Set Thermocycler to this program:</p>
<ul>
<li>Incubate at 25°C for 10 min.</li>
<li>Incubate at 42°C for 90 min.</li>
<li>10 cycles of 50°C for 2 min and 42°C for 2 min.</li>
<li>Deactivate at 70°C for 10 min.</li>
<li>Hold at 4°C.</li>
</ul>
</li>
<li>
<p>Purify cDNA with <a href="https://www.cytivalifesciences.com/en/us/shop/molecular-and-immunodiagnostics/pcr-cleanup-and-size-selection/illustra-microspin-g-50-columns-p-00056?s_kwcid=AL!14612!3!675381046559!!!g!!&amp;dtid=semp_google_20584436769_152358921965&amp;ps_kw=&amp;extcmp=G-SE-PAID-CY23072-GLOBAL-ALL-Research-matters-Google-Ads-Microspin-G-50-Columns&amp;adgrp=&amp;gclid=Cj0KCQjw4bipBhCyARIsAFsieCx4oBgd-ebx6sMs5pbFPAENOKxupyYOSkLWZ0s7ugzSo3WgdScv5nAaAkZpEALw_wcB#related-documents">G-50 column</a></p>
</li>
</ol>
<h2 id="more-information">More Information:</h2>
<ul>
<li><a href="https://tools.thermofisher.com/content/sfs/manuals/superscriptII_pps.pdf">First-Stand cDNA Synthesis Using SuperScript II RT</a></li>
</ul>
<h1 id="second-strand-synthesis-2-hours">Second Strand Synthesis (~2 hours)</h1>
<p>From first strand synthesis product, a mix of enzymes polymerizes a second
cDNA from the first cDNA of a DNA-RNA hybrid, then digests away the RNA.</p>
<h2 id="before-starting">Before starting:</h2>
<ul>
<li><a href="map-rt-dms.md">first strand synthesis</a></li>
<li><a href="../purification/spri-bead-purification.md">cDNA purification</a></li>
</ul>
<h2 id="materials">Materials:</h2>
<ul>
<li>NEBNext second strand synthesis enzyme mix</li>
<li>NEBNext second strand synthesis 10x buffer</li>
<li>Fresh 80% EtOH</li>
</ul>
<h2 id="protocol">Protocol:</h2>
<p><strong>Always keep 2nd Strand Enzyme cold (below 16°C)</strong></p>
<ol>
<li>
<p>Into a tube combine:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Reagent</th>
<th style="text-align:left">Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">sscDNA</td>
<td style="text-align:left">20 μL</td>
</tr>
<tr>
<td style="text-align:left">nucleotide-free water</td>
<td style="text-align:left">48 μL</td>
</tr>
<tr>
<td style="text-align:left">10X 2nd Strand Reaction Buffer</td>
<td style="text-align:left">8 μL</td>
</tr>
<tr>
<td style="text-align:left">2nd Strand Enzyme Mix</td>
<td style="text-align:left">4 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:left"><strong>80 μL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Mix by pipetting.</p>
</li>
<li>
<p>Incubate at 16°C for 60 min.</p>
</li>
</ol>
<h2 id="next-steps">Next Steps:</h2>
<ul>
<li><a href="../purification/spri-bead-purification.md">Purify DNA</a></li>
<li><a href="../assays/hs-dna.md">HS DNA Qubit assay</a></li>
<li><a href="nextera-xt.md">Nextera XT</a></li>
</ul>
<h1 id="dna-library-prep-adapter-ligation---nebnext-ultra-ii-2-hours">DNA Library Prep (Adapter Ligation) - NEBNEXT Ultra II (~2 hours)</h1>
<h2 id="materials">Materials:</h2>
<ul>
<li>dsDNA (500 pg - 1 μg)</li>
<li>NEBNext Ultra II End Prep Enzyme Mix</li>
<li>NEBNext Ultra II End Prep Reaction Buffer</li>
<li>NEBNext Ultra II Ligation Master Mix</li>
<li>NEBNext Ligation Enhancer</li>
<li>NEBNext Ultra II Q5 Master Mix</li>
<li>NEBNext Oligo Kit</li>
<li>80% Ethanol</li>
</ul>
<h2 id="protocol">Protocol:</h2>
<ol>
<li>
<p>Prepare Ends.</p>
<ul>
<li>
<p>Add the following components into a PCR tube:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Components</th>
<th style="text-align:right">Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">dsDNA</td>
<td style="text-align:right">50 μL</td>
</tr>
<tr>
<td style="text-align:left">End Prep Reaction Buffer</td>
<td style="text-align:right">7 μL</td>
</tr>
<tr>
<td style="text-align:left">End Prep Enzyme Mix</td>
<td style="text-align:right">3 μL</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:right"><strong>60 μL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Mix thoroughly and spin down.</p>
</li>
<li>
<p>Inbucate in thermal cycler with lid set to &gt; 75 °C:</p>
<table>
<thead>
<tr>
<th style="text-align:left">Step</th>
<th style="text-align:center">Time (min.)</th>
<th style="text-align:center">Temperature (°C)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Prep ends</td>
<td style="text-align:center">30</td>
<td style="text-align:center">20</td>
</tr>
<tr>
<td style="text-align:left">Denature</td>
<td style="text-align:center">30</td>
<td style="text-align:center">65</td>
</tr>
<tr>
<td style="text-align:left">End</td>
<td style="text-align:center">HOLD</td>
<td style="text-align:center">4</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
<li>
<p>Adapter Ligation. (Adds ~60 nt)</p>
<ul>
<li>
<p>Dilute adaptors based on input DNA quantity (from NEBNext Oligo Kit):</p>
<table>
<thead>
<tr>
<th style="text-align:left">Input DNA</th>
<th style="text-align:center">Adpator Dilution</th>
<th style="text-align:center">Working [Adapter]</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">101-1000 ng</td>
<td style="text-align:center">No Dilution</td>
<td style="text-align:center">15 μM</td>
</tr>
<tr>
<td style="text-align:left">5 - 100 ng</td>
<td style="text-align:center">1:10</td>
<td style="text-align:center">1.5 μM</td>
</tr>
<tr>
<td style="text-align:left">0.5 - 5 ng</td>
<td style="text-align:center">1:25</td>
<td style="text-align:center">0.6 μM</td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Add the following components directly into the End Prep Reaction Mixture:</p>
<ul>
<li><strong>NOTE: Do NOT premix Adapter with Master Mix or Ligation Enhancer.</strong></li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">COMPONENT</th>
<th style="text-align:right">VOLUME</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">End Prep Reaction Mixture</td>
<td style="text-align:right">60 µl</td>
</tr>
<tr>
<td style="text-align:left">NEBNext Adaptor (diluted)</td>
<td style="text-align:right">2.5 µl</td>
</tr>
<tr>
<td style="text-align:left">NEBNext Ultra II Ligation Master Mix</td>
<td style="text-align:right">30 µl</td>
</tr>
<tr>
<td style="text-align:left">NEBNext Ligation Enhancer</td>
<td style="text-align:right">1 µl</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:right"><strong>93.5 µL</strong></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Mix thoroughly and spin down.</p>
</li>
<li>
<p>Incubate at 20 °C for 15 minutes with lid heater off.</p>
</li>
<li>
<p>Add 3 µl of (red) USER enzyme.</p>
</li>
<li>
<p>Mix throughly.</p>
</li>
<li>
<p>Incubate at 37 °C for 15 minutes with the lid heater set to &gt; 47 °C</p>
</li>
</ul>
</li>
<li>
<p>Cleanup of Adaptor-ligated DNA - SPRI beads</p>
<p><strong>Bead Ratio: 1.4x</strong></p>
<ol>
<li>Beads added to reaction mixture (1.4X)</li>
<li>Incubate at room temp. for 5 min.</li>
<li>Place in magnetic holder for 5 min.</li>
<li>Remove most of the supernatant</li>
<li>Wash beads with 80% ethanol (30 seconds) twice.</li>
<li>Add 17 μL of water, resuspend pellet, and incubate for 5 minutes.</li>
<li>Remove and keep supernatant.</li>
</ol>
</li>
<li>
<p>PCR Amplification (Adds ~60 nt)</p>
<ul>
<li>Add the following components to a PCR strip:</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">COMPONENT</th>
<th style="text-align:right">VOLUME</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Adaptor Ligated DNA Fragments</td>
<td style="text-align:right">15 µl</td>
</tr>
<tr>
<td style="text-align:left">NEBNext Ultra II Q5 Master Mix</td>
<td style="text-align:right">25 µl</td>
</tr>
<tr>
<td style="text-align:left">Index Primer/i7 Primer</td>
<td style="text-align:right">5 µl</td>
</tr>
<tr>
<td style="text-align:left">Universal PCR Primer/i5 Primer</td>
<td style="text-align:right">5 µl</td>
</tr>
<tr>
<td style="text-align:left"><strong>TOTAL</strong></td>
<td style="text-align:right"><strong>50 µl</strong></td>
</tr>
</tbody>
</table>
<ul>
<li>Mix thoroughly and spin down.</li>
<li>Choose number of PCR cycles using the table below:</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:right">INPUT DNA</th>
<th style="text-align:center"># OF CYCLES (YIELD ~100 ng)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right">100 ng</td>
<td style="text-align:center">3</td>
</tr>
<tr>
<td style="text-align:right">50 ng</td>
<td style="text-align:center">3–4</td>
</tr>
<tr>
<td style="text-align:right">10 ng</td>
<td style="text-align:center">6–7</td>
</tr>
<tr>
<td style="text-align:right">5 ng</td>
<td style="text-align:center">7–8</td>
</tr>
<tr>
<td style="text-align:right">1 ng</td>
<td style="text-align:center">9–10</td>
</tr>
<tr>
<td style="text-align:right">0.5 ng</td>
<td style="text-align:center">10–13</td>
</tr>
</tbody>
</table>
<ul>
<li>Perform PCR amplification using the following PCR cycling conditions:</li>
</ul>
<table>
<thead>
<tr>
<th style="text-align:left">STEP</th>
<th>PCR Cycle Step</th>
<th>TEMP</th>
<th>TIME</th>
<th>CYCLES</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Initial Denaturation</td>
<td></td>
<td>98 °C</td>
<td>30 seconds</td>
<td>1</td>
</tr>
<tr>
<td style="text-align:left">PCR cycles:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td>Denaturation</td>
<td>98 °C</td>
<td>10 seconds</td>
<td>3-15</td>
</tr>
<tr>
<td style="text-align:left"></td>
<td>Annealing/Extension</td>
<td>65 °C</td>
<td>75 seconds</td>
<td>3-15</td>
</tr>
<tr>
<td style="text-align:left">Final Extension</td>
<td></td>
<td>65 °C</td>
<td>5 minutes</td>
<td>1</td>
</tr>
<tr>
<td style="text-align:left">HOLD</td>
<td></td>
<td>4 °C</td>
<td>Hold</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Cleanup of PCR Reaction - SPRI beads</p>
<p><strong>Bead Ratio: 1.2x</strong></p>
<ol>
<li>Beads added to reaction mixture (1.2X)</li>
<li>Incubate at room temp. for 5 min.</li>
<li>Place in magnetic holder for 5 min.</li>
<li>Remove most of the supernatant</li>
<li>Wash beads with 80% ethanol (30 seconds) twice.</li>
<li>Add 20 μL of water, resuspend pellet, and incubate for 5 minutes.</li>
<li>Remove and keep supernatant.</li>
</ol>
</li>
</ol>
<h2 id="notes">Notes:</h2>
<ul>
<li>12 cycles of PCR used</li>
<li>Lengths are insert + adaptor (60 nt) + primers (60 nt)</li>
</ul>
<h2 id="next-steps">Next Steps:</h2>
<ul>
<li>Qubit</li>
<li>Bioanalyzer</li>
</ul>
<h1 id="qubit-05-hours">Qubit (~0.5 hours)</h1>
<p>Quantify the dscDNA product from second strand synthesis via Qubit HS DNA assay.</p>
<h2 id="material">Material</h2>
<ul>
<li><a href="https://www.thermofisher.com/order/catalog/product/Q33230">Qubit 1X dsDNA HS kit</a>
<ul>
<li>Qubit tubes</li>
<li>1X working solution</li>
<li>Standards 1 &amp; 2</li>
</ul>
</li>
</ul>
<h2 id="protocol">Protocol</h2>
<ol>
<li>Create standard 1 &amp; 2:
<ul>
<li>Add 190 μL of working solution into Qubit tube</li>
<li>Add 10 μL of standard</li>
<li>Equalibrate to RT</li>
</ul>
</li>
<li>Create samples:
<ul>
<li>Add 196 μL of working solution into Qubit tube</li>
<li>Add 4 μL of cDNA</li>
<li>Equalibrate to RT</li>
</ul>
</li>
<li>Incubate for 2 minutes before reading concentrations</li>
</ol>
<h2 id="results">Results</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Sample ID</th>
<th style="text-align:center">[dscDNA] ng/μL</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
</tr>
</tbody>
</table>
<h1 id="bioanalyzer-1-hour">Bioanalyzer (~1 hour)</h1>
<p>Determine the average length of nextera library.</p>
<h2 id="materials">Materials</h2>
<ul>
<li>High sensitivity DNA reagants</li>
<li>High sensitivity DNA chip</li>
<li>Chip priming station</li>
</ul>
<h2 id="protocol">Protocol</h2>
<ul>
<li>Follow Agilent <a href="https://www.agilent.com/cs/library/usermanuals/Public/G2938-90321_SensitivityDNA_KG_EN.pdf">protocol</a></li>
</ul>
<h2 id="results">Results</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Sample ID</th>
<th style="text-align:center">Avg. bp length</th>
<th style="text-align:right">% of total</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
</tbody>
</table>
<h1 id="pool-and-dilute-library-1-hour">Pool and dilute library (~1 hour)</h1>
<p>Dilute sample to the same molarity and pool for sequencing.</p>
<h2 id="material">Material</h2>
<ul>
<li><a href="https://github.com/Weeks-UNC/2023_Veenbaas_PNAS_fpocketR_Frag-MaP/blob/main/Frag-MaP/protocol/Illumina-mix-calculator.xlsx">Illumina-mix-calculator.xlsx</a></li>
<li><a href="https://github.com/Weeks-UNC/2023_Veenbaas_PNAS_fpocketR_Frag-MaP/blob/main/Frag-MaP/protocol/SAMPLE_SHEET.xlsx">Sample_Sheet.xlsx</a></li>
</ul>
<h2 id="target-library-concentration-for-illumina-v3-chemistry">Target library concentration for illumina v3 chemistry</h2>
<p>2.00 nM [library] -&gt; 10 pM [final]</p>
<h2 id="protocol">Protocol</h2>
<ol>
<li>
<p>Input data into illumina-mix-calculator.xlsx</p>
<ul>
<li>Avg. bp length (Bioanalyzer)</li>
<li>ng/μL (Qubit)</li>
<li>% Non-dimer (bioanalyzer)</li>
<li>Target library concentration (nM)</li>
</ul>
<p><strong>NOTE: Recomended minimum volume: 3 μL</strong></p>
</li>
<li>
<p>Pool sample library using the volumes calculated by illumina-mix-calculator.xlsx</p>
</li>
<li>
<p>Quantify the concentration of the pooled sample library using Qubit</p>
</li>
<li>
<p>Dilute the pooled library to the final concentration (ng/μL) calculated by illumina-mix-calculator.xlsx</p>
<table>
<thead>
<tr>
<th style="text-align:left">Sample ID</th>
<th style="text-align:center">[non-dimer] (nM)</th>
<th style="text-align:right">Amount to pool (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:left"></td>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
</tr>
</tbody>
</table>
</li>
<li>
<p>Verify concentration via Qubit.</p>
<table>
<thead>
<tr>
<th style="text-align:center">Sample ID</th>
<th style="text-align:right">ng/μL</th>
<th style="text-align:right">nM</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
<tr>
<td style="text-align:center"></td>
<td style="text-align:right"></td>
<td style="text-align:right"></td>
</tr>
</tbody>
</table>
</li>
</ol>
<h2 id="next-steps">Next Steps</h2>
<p>MiSeq</p>
<h1 id="miseq-2-hours">MiSeq (~2 hours)</h1>
<p>Normalized DNA sequencing libraries are pooled, denatured, diluted, loaded into a MiSeq reagent kit, and ran on the MiSeq</p>
<ol>
<li>
<p>Post-run wash (if not already completed)</p>
<ul>
<li>After the run, discard waste, reagent cartridge and PR2 bottle
<ul>
<li>Formamide must be removed from the reagent cartridge (Well #8)</li>
</ul>
</li>
<li>Fill an empty PR2 bottle with 500 mL 0.5% Tween 20:
<ul>
<li>25 mL 10% Tween 20</li>
<li>475 MilliQ water</li>
</ul>
</li>
<li>From PR2 bottle, put 5 mL of 0.5% Tween 20 into each well of the wash tray</li>
<li>Load wash tray, wash bottle, and waste bottle</li>
<li>Start post-run wash</li>
</ul>
</li>
<li>
<p>Reboot computer and clear disk space</p>
<ul>
<li>Make drive has at least 100 GB free</li>
<li>Shut-down windows through the start menu</li>
<li>Once complete, shut off power to the device for 30 seconds</li>
<li>Turn on power switch, device will reboot</li>
</ul>
</li>
<li>
<p>Thaw reagent cartridge</p>
<ul>
<li>Open MiSeq Reagent Kit Box 1 of 2 (stored at -20)</li>
<li>Thaw HT1 buffer on ice (or RT if impatient, then move to ice)</li>
<li>Copy the serial number from the reagent cartridge (MS#######-#00V#)</li>
<li>Thaw reagent cartridge in a room temperature water bath (~1 hr)</li>
<li>Once completely thawed, store at 4C until loading</li>
</ul>
</li>
<li>
<p>Pool, denature, and dilute libraries</p>
<ul>
<li>Prepare a fresh dilution of 0.2 N NaOH by mixing
<ul>
<li>200 uL 1.0 N NaOH</li>
<li>800 uL nuclease free water</li>
</ul>
</li>
<li>Dilute samples to 2.0 nM and combine in the desired ratio</li>
<li>In a 1.7 mL tube, combine
<ul>
<li>5 uL 2.0 nM library</li>
<li>5 uL 0.2 N NaOH</li>
</ul>
</li>
<li>Briefly vortex and quick spin down (280 x g for 1 min)</li>
<li>Incubate at room temp for at least 5 min</li>
<li>Add 990 uL chilled HT1</li>
<li>Sample library is at 10 pM in HT1 with 1mM NaOH</li>
</ul>
</li>
<li>
<p>Clean flow cell</p>
<ul>
<li>Remove flow cell from MiSeq Reagent Kit Box 2 of 2 (4 C), handle by the edges</li>
<li>Wash flow cell with water</li>
<li>Carefully clean glass surface with lens paper</li>
<li>Repeat until surface is free of smudges</li>
<li>Wrap in lens paper until needed</li>
</ul>
</li>
<li>
<p>Load cartridge and start run (load within 1 hour of denaturing samples)</p>
<ul>
<li>Flip cartridge 10x to ensure proper mixing</li>
<li>Using a 1000 uL tip, poke open the hole for the sample</li>
<li>Load 600 uL sample (avoid air bubbles), gently tap cartridge so libraries pool at bottom</li>
<li>Follow the on-screen directions to insert the flow cell, reagent cartridge, and PR2 bottle</li>
</ul>
</li>
<li>
<p>Post-run wash (perform ASAP, within 24 hours of run completion)</p>
<ul>
<li>After the run, discard waste, reagent cartridge and PR2 bottle
<ul>
<li>Formamide must be removed from the reagent cartridge (Well #8)</li>
</ul>
</li>
<li>Fill an empty PR2 bottle with 500 mL 0.5% Tween 20:
<ul>
<li>25 mL 10% Tween 20</li>
<li>475 MilliQ water</li>
</ul>
</li>
<li>From PR2 bottle, put 5 mL of 0.5% Tween 20 into each well of the wash tray</li>
<li>Load wash tray, wash bottle, and waste bottle</li>
<li>Start post-run wash</li>
</ul>
</li>
</ol>
<h3 id="sample-details">Sample details:</h3>
<ul>
<li>Final library concentration should be ~10 pM</li>
<li>Cluster density should be ~1000-1200</li>
</ul>
<h3 id="kit-details">Kit details:</h3>
<table>
<thead>
<tr>
<th style="text-align:center">Box</th>
<th style="text-align:center">Ref</th>
<th style="text-align:center">Lot</th>
<th style="text-align:center">Exp</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">1</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
<tr>
<td style="text-align:center">2</td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
<td style="text-align:center"></td>
</tr>
</tbody>
</table>

</body>
</html>
